<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeCalculatorWidget</class>
 <widget class="QWidget" name="VolumeCalculatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Volume Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_selection">
     <property name="title">
      <string>Region Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QPushButton" name="pushButton_selectRegion">
        <property name="text">
         <string>Select Region</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_updatePolygon">
        <property name="text">
         <string>Update Polygon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clearPoints">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_points">
     <property name="title">
      <string>Selected Points</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget_points">
        <column>
         <property name="text">
          <string>Longitude</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Latitude</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Altitude (m)</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_calculation">
     <property name="title">
      <string>Volume Calculation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_calculateVolume">
        <property name="text">
         <string>Calculate Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_volume">
        <property name="text">
         <string>Volume (mÂ³):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_volume">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
